### ISSUE TITLE
Strange things happen when minimizing circuit

### ISSUE THREAD

**Issue by a-anjos** [logisim-evolution/logisim-evolution#1278]
When going to the tab "Minimized" in the "Combinatorial Analisys" window, it doesn't work. I attach an image showing what happens.
![image](https://user-images.githubusercontent.com/42868600/140085467-914e9c7a-af04-4420-bde6-b5eec4f0f092.png)
I'm using Version 3.7.1 on Debian Bullseye.
Cheers,
António

PS:

```
Exception in thread "AWT-EventQueue-0" java.lang.RuntimeException: Row 0 is outside range of 4 rows.
	at com.cburch.logisim.analyze.gui.KarnaughMapPanel.label(KarnaughMapPanel.java:563)
	at com.cburch.logisim.analyze.gui.KarnaughMapPanel.drawNumberedHeader(KarnaughMapPanel.java:606)
	at com.cburch.logisim.analyze.gui.KarnaughMapPanel.paintKmap(KarnaughMapPanel.java:525)
	at com.cburch.logisim.analyze.gui.KarnaughMapPanel.paintComponent(KarnaughMapPanel.java:478)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1119)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JViewport.paint(JViewport.java:736)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1128)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:952)
	at java.desktop/javax.swing.JComponent.paintToOffscreen(JComponent.java:5318)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBufferedImpl(RepaintManager.java:1644)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBuffered(RepaintManager.java:1619)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paint(RepaintManager.java:1557)
	at java.desktop/javax.swing.BufferStrategyPaintManager.paint(BufferStrategyPaintManager.java:262)
	at java.desktop/javax.swing.RepaintManager.paint(RepaintManager.java:1324)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1105)
	at java.desktop/java.awt.GraphicsCallback$PaintCallback.run(GraphicsCallback.java:39)
	at java.desktop/sun.awt.SunGraphicsCallback.runOneComponent(SunGraphicsCallback.java:75)
	at java.desktop/sun.awt.SunGraphicsCallback.runComponents(SunGraphicsCallback.java:112)
	at java.desktop/java.awt.Container.paint(Container.java:2006)
	at java.desktop/java.awt.Window.paint(Window.java:3954)
	at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:877)
	at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:849)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:849)
	at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:824)
	at java.desktop/javax.swing.RepaintManager.prePaintDirtyRegions(RepaintManager.java:773)
	at java.desktop/javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1885)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
appVersionJvm
```

**Comment by BFH-ktt1** [comment]
This is a duplicate and fixed in #1266. Please try the nightly build and see if it solves the problem. Closing

**Comment by a-anjos** [comment]
The night build has the issue solved. Thanks @BFH-ktt1
